foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -215,7 +218,8 @@ def retry_lost_processor_jobs() -> None:
         start_time=None,
         end_time=None,
         created_at__lt=minimum_creation_time
-    )
+        # TEMPORARY for Jackie's grant
+    ).exclude(pipeline_applied=ProcessorPipeline.NONE.value)
```

Comment:
Yes it is, although you actually just provided one example of why this is useful. `pipeline_applied="None"` is actually wrong, it would have to be `pipeline_applied="NONE"`. By avoiding the use of magic strings I guarantee that they are all consistent everywhere. Basically its the same argument as ever for using Enumerated Types.